getThemeYaml.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
```

Comment:
Better to remove this file :) this was example of usage on `travix-themes`. Which is unrelated to ui-kit itself.